packages/docusaurus-theme-classic/src/theme/hooks/useWindowSize.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {useEffect, useState} from 'react';
+
+const desktopThresholdWidth = 996;
+const desktopSize = 'desktop';
+const mobileSize = 'mobile';
+
+function useWindowSize() {
+  const isClient = typeof window === 'object';
+
+  function getSize() {
+    if (isClient) {
```

Comment:
I think it's !isClient instead :)

Reply:
Indeed, my bad, went to quickly on that one.